### ISSUE TITLE
Bot ignores links in Reddit text posts.

### ISSUE THREAD

**Issue by pfandzelter** [SolarLiner/Subreddit-Linker-Bot#1]
The bot "fixes" links where the title links to subreddits, although the body of the text post links to them as well ([e.g. Here](http://www.reddit.com/r/trendingsubreddits/comments/2yf7yl/trending_subreddits_for_20150309_rmagiceye/)). This is not a big issue but may be a bit inconvenient (see the downvotes). I suggest scanning the subreddit for links before posting.

**Comment by SolarLiner** [comment]
Yeah, seeing those I thought that i maybe "underestimated" the uselessness on certain type of posts.
I'll see what I can do on that, thanks.
